itest/rust/src/object_tests/object_test.rs

```diff
@@ -911,12 +911,16 @@ pub mod object_test_gd {
         i: i64,
     }
 
-    #[derive(GodotClass, Debug)]
-    #[class(init, base=RefCounted)]
-    struct ObjectTest;
+    mod nested {
+        use godot::prelude::*;
+        #[derive(GodotClass, Debug)]
+        #[class(init, base=RefCounted)]
+        pub(super) struct ObjectTest;
+    }
+    use nested::ObjectTest;
```

Comment:
I nested the ObjectTest struct in its own little mod to make sure that nothing broke if I used a more qualified path in the impl block.